src/index.js

```diff
@@ -1,24 +1,20 @@
+import addScore from './modules/addScores.js';
+import getScores from './modules/getScores.js';
 import './style.css';
-import addLi from './modules/add-li.js';
 
-const leaderboardData = {
-  result: [
-    {
-      user: 'John Doe',
-      score: 42,
-    },
-    {
-      user: 'Peter Parker',
-      score: 35,
-    },
-    {
-      user: 'Wonder Woman',
-      score: 50,
-    },
-  ],
-};
+const scoresList = document.querySelector('.scores-list');
+const submitButton = document.querySelector('.submit-button');
+const refreshButton = document.querySelector('.refresh');
+const message = document.querySelector('.message');
 
-leaderboardData.result.forEach((element) => {
-  const text = `${element.user}: ${element.score}`;
-  addLi(text);
+refreshButton.addEventListener('click', (e) => {
+  e.preventDefault();
+  getScores(scoresList);
+});
```

Comment:
Nice work with your refresh button! It is working as intended. Please note that when the user enters the page, the score list is empty:
![image](https://user-images.githubusercontent.com/42799579/197360546-a8de2778-596a-4e7c-8c39-f13efc4996de.png)

It would be a great addition if you also fetch this data when loading the page, not only after clicking on the refresh button. This would display the score list right after loading the elements 😉.